### PR TITLE
[Cloud Functions] Update npm and Node.js versions and tweak README.md

### PR DESCRIPTION
**Pull request by gino-m**
Hi @Tiyarah, mind taking a look? 

The npm dep version updates resolve some known security vulnerabilities.

### CODE REVIEW

**Comment by gino-m** [comment]
@Tiyarah gentle ping :)

**Comment by Tiyarah** [comment]
LGTM.

File: functions/README.md [hunk 1]
```diff
@@ -46,77 +44,81 @@ cloud functions.
...
-    $ firebase use --add <project-name>
-    ```
-  
-    Where `<project-name>` is the name of a project associated with your firebase
-    account.
+   ```
+   $ firebase use --add <project-name>
+   ```
+
+   Where `<project-name>` is the name of a project associated with your Firebase
```
Comment (Tiyarah) [inline functions/README.md:74]: Perhaps it is worth mentioning that the developer can reuse one of the ground projects from a list or something?

Comment (gino-m) [reply functions/README.md:74]: This note refers to Firebase projects, not Ground projects. Developers will need to set up a project before going through this process; we should probably document this somewhere. Files #507 to track.

**Review by Tiyarah — APPROVED** [review]

**Pull request merged**